theme/styles/snippets/product-grid-item.scss

```diff
@@ -44,6 +44,11 @@
     letter-spacing: 0.0275rem;
   }
 
+  &__price {
```

Comment:
Not sure how much familiarity you have with our approach to styling, but we try to use utility/atomic classes as much as possible, usually only writing custom css/sass when complex styling is required, or when there is lots of variation per breakpoint. 

In this case, you could use:

pl1_5 (which maps to `padding-left: 1.5rem`)
text-left (which maps to `text-align: left`)

Reply:
I think I went for the custom css here because it appeared in two places but being consistent makes sense